src/familiar/barfFamiliar.ts

```diff
@@ -164,7 +164,7 @@ export function barfFamiliar(): Familiar {
 
   if (viableMenu.length === 0) return meatFamiliar();
 
-  const best = viableMenu.reduce((a, b) => (totalFamiliarValue(a) > totalFamiliarValue(b) ? a : b));
+  const best = maxBy(viableMenu, totalFamiliarValue);
```

Comment:
this is more of a general comment than a comment about this explicitly. 

one small issue i have here -- and this is less an issue in vscode (which syntax highlights correctly) and more on github proper -- is that totalFamiliarValue isn't highlighted as a function on github. 

syntax-wise does it also work if you do totalFamiliarValue() or does it have to exclude the parens? if so i might request doing () to make it clear even on github review that this is a function. this is not a major bug in my craw or anything but it would slightly help cross-platform syntax highlighting.

Reply:
you would have to do `(x) => totalFamiliarValue(x)`, which I personally think is uglier but we each have our own feelings

Reply:
`totalFamiliarValue()` doesn't exist, because this function takes an input. When you do give it an input, it's a number; we want the function itself, and that means either passing just `totalFamiliarValue` or using arrows (or the `function` keyword, I guess) to construct a function using it. Which always feels a little silly to me

Reply:
that's fine then. no big issue from me if it's not doable it's just an annoying syntax thing with github